src/components/map-marker/map-marker.tsx

```diff
@@ -62,16 +69,38 @@ export class MapMarker implements GxComponent {
    */
   @Event() gxMapMarkerDeleted: EventEmitter;
 
+  private setPopup() {
+    const popupHtml = this.element.querySelector("[class='popupHtml']");
+    if (popupHtml.innerHTML) {
+      const marginProportion = 83 / 100;
```

Comment:
Magic numbers 👀

Reply:
No no 😅
```marginProportion``` is a variable to avoid that magic number ```83/100```
I decided to leave it like that because is more mathematically intuitive.
Don't need to be a super mathy person to realize that ```83/100``` is 83% of something, and I think that ```83/100``` is better than type just ```0.83```...  it's more mathematically intuitive...

Wht do u think that i shall do?

Reply:
.83 is fine.
You could also leave this constant value outside the class, for example:

const MAX_POPUP_SIZE_FACTOR = .83;